miden-lib/asm/miden/account.masm

```diff
@@ -134,11 +131,10 @@ end
 #! - OLD_MAP_VALUE is the old value at KEY.
 export.set_map_item
     syscall.set_account_map_item
-    # => [OLD_MAP_ROOT, OLD_MAP_VALUE]
-
-    # prepare stack for return
-    push.0 movdn.9
     # => [OLD_MAP_ROOT, OLD_MAP_VALUE, 0]
+
+    movup.8 drop
+    # => [OLD_MAP_ROOT, OLD_MAP_VALUE]
 end
```

Comment:
Great catch! Could you extract this fix into a separate PR? I'd like to merge it before releasing `miden-base` v0.4 (which I'm hoping to do today).

Reply:
https://github.com/0xPolygonMiden/miden-base/pull/785